conan/api/subapi/upload.py

```diff
@@ -57,6 +59,40 @@ def upload(self, package_list, remote):
         executor = UploadExecutor(app)
         executor.upload(package_list, remote)
 
+    def full_upload(self, package_list, remote, enabled_remotes, check_integrity=False, force=False,
```

Comment:
It would be nice to add a docstring here, because this is used for the docs to add documentation for the public API methods.

Reply:
Suggestions for a better name? I didn't love it, but also not better ideas. I thought repurposing the old ``upload()`` to mean this full one, and rename the old one to ``execute_upload`` or ``upload_transfer`` or something like that, but I am afraid that this could break users even if this is not public/stable

Reply:
I could not think of a better name, but I think I would use `upload_full` instead of `full_upload` so it's easier to find.

Reply:
Changed to ``upload_full``